DesktopEditor/fontengine/ApplicationFonts.cpp

```diff
@@ -1638,7 +1638,7 @@ void CApplicationFonts::Initialize(bool bIsCheckSelection)
 	InitFromReg();
 #endif
 
-#if defined(_LINUX) && !defined(_MAC) && !defined(__ANDROID__)
+#if ( defined(_LINUX) || defined(__FreeBSD__) ) && !defined(_MAC) && !defined(__ANDROID__)
 	m_oList.LoadFromFolder(L"/usr/share/fonts");
```

Comment:
/usr/share/fonts  doesn't exist on FreeBSD

Reply:
you're right, it would be great to get them from /usr/local/share/fonts

Reply:
I updated the code in order to use /usr/local/share/fonts.